code/game/machinery/computer/teleporter.dm

```diff
@@ -189,7 +189,7 @@
 			return
 		set_teleport_target(targets[desc])
 		var/turf/target_turf = get_turf(targets[desc])
-		log_game("[key_name(user)] has set the teleporter target to [targets[desc]] at [AREACOORD(target_turf)]")
+		user.log_message("set the teleporter target to [targets[desc]] at [AREACOORD(target_turf)]", LOG_GAME)
```

Comment:
```suggestion
		user.log_message("set the teleporter target to [targets[desc]]", LOG_GAME)
```

Reply:
(If you're removing this you should also remove the line above it that gets the turf)